browser/help/topics/v3-ht-annotations.md

```diff
@@ -165,22 +165,22 @@ The gnomAD v3 Hail Table annotations are defined below:
-  - **pangolin**:
-    - **largest_ds**: The largest delta score of a variant involved in 2 splicing consequences across genes if a variant was predicted on multiple genes. It ranges from -1 to 1, positive are for splice loss variants and > 0 are for splice-gain variants. Pangolin also proposed a cutoff of >|0.2| for the predicted increase or decrease in splice site usage. 
+- **in_silico_predictors**: Variant prediction annotations. We put together the prediction scores from multiple in silico predictors for variants that are predicted to be missense, impacting protein function, evolutionarily conserved or splice-altering. We focus on MANE Select and canonical transcripts if the prediction scores are available for multiple transcripts.
+  - **revel**: Rare Exome Variant Ensemble Learner (REVEL) [score](https://sites.google.com/site/revelgenomics/about). An ensemble score for predicting the pathogenicity of missense variants (based on 13 other variant predictors).
+    - **revel_max**: Score predicting the pathogenicity of missense variants with a range of 0 to 1. Variants with higher scores are predicted to be more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
+  - **primateai_3d**: A deep neural network-trained [score](https://www.science.org/doi/10.1126/science.abn8197) for classifying pathogenicity of missense variants.
+    - **primateai-3d_max**: Score ranging from 0 to 1 predicting the deleteriousness of missense variants. Higher scores indicate a variant is more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
+  - **cadd**: Combined Annotation Dependent Deletion [score](https://cadd.gs.washington.edu/) used to predict deleteriousness of SNVs and indels.
+    - **phred**: CADD Phred-like scaled C-scores ranging from 1 to 99 based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome. Larger values indicate increased predicted deleteriousness.
+    - **raw_score**: Unscaled CADD scores indicating whether a variant is likely to be "observed" (negative values) vs "simulated" (positive values). Larger values indicate increased predicted deleteriousness.
+  - **sift**: Sorting Tolerant From Intolerant (SIFT) [score](https://www.nature.com/articles/nprot.2009.86) used to predict the impact of SNVs on protein function.
+    - **sift_max**: Scaled probability of the amino acid substitution being tolerated, ranging from 0 to 1. Scores below 0.05 are predicted to impact protein function.  A normalized probability of observing the new amino acid at that position to affect protein function, and ranges from 0 to 1. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
```

Comment:
```suggestion
    - **sift_max**: Scaled probability of the amino acid substitution being tolerated, ranging from 0 to 1. Scores below 0.05 are predicted to impact protein function. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
```